plasmapy/physics/transport.py

```diff
@@ -1,17 +1,78 @@
 """Functions to calculate transport coefficients."""
 
-from astropy import units
+from astropy import u
 import numpy as np
 from ..utils import check_quantity, _check_relativistic
-from ..constants import (m_p, m_e, c, mu0, k_B, e, eps0, pi, h, hbar)
-from ..atomic import (ion_mass, charge_state)
-from .parameters import Debye_length
-
+from ..constants import (m_p,
+                         m_e,
+                         c,
+                         mu0,
+                         k_B,
+                         e,
+                         eps0,
+                         pi,
+                         h,
+                         hbar,
+                         )
+from ..atomic import (ion_mass,
+                      charge_state,
+                      )
+from .parameters import (Debye_length,
+                         )
+from .quantum import (Wigner_Seitz_radius,
+                      thermal_deBroglie_wavelength,
+                      chemical_potential,
+                      )
+from ..math import Fermi_integral
+
+
+
+
+def boilerPlate(T, particles, V):
```

Comment:
If we're keeping this function (and we very well may want to) under the current name, let's at least rename it to `_boilerPlate` with an underscore so it's "considered less useful outside here" :smiley:

Reply:
done